packages/turf-concave/index.js

```diff
@@ -15,10 +15,9 @@ var distance = require('@turf/distance');
  * @param {FeatureCollection<Point>} points input points
  * @param {number} maxEdge the size of an edge necessary for part of the
  * hull to become concave (in miles)
- * @param {string} units used for maxEdge distance (miles or kilometers)
+ * @param {String} [units=kilometers] can be degrees, radians, miles, or kilometers
```

Comment:
string (rather than String) is canonical here, since primitive strings aren't actually full instances of the String class